sanic/app.py

```diff
@@ -111,7 +112,7 @@ def decorator(listener):
 
     # Decorator
     def route(self, uri, methods=frozenset({'GET'}), host=None,
-              strict_slashes=False, stream=False, version=None):
+              strict_slashes=None, stream=False, version=None):
```

Comment:
Instead of `strict_slashes=None` I think it's better to do `strict_slashes=self.strict_slashes` since it's more compact. What do you think?